### DISCUSSION TITLE
[perf] Dashboard loading time is 5+ mins with multiple tabs, tabs in tabs, native filters

### DISCUSSION THREAD

**Discussion by rosemarie-chiu** [apache/superset#19199] [Ideas]
I have to trimmed out 2+mins on the video for it to be under 10MB. 

1. It took 10 secs to respond to click to 2nd tab (where the contents are)
2. 10 secs to respond to click to tab in tab
3. Content shows up 2 mins after I end this video, so about 3+mins to show up all content on that tab

https://user-images.githubusercontent.com/52086618/125660894-9143be50-9546-4480-8e80-2058a3646fbe.mov

**Comment by junlincc** [comment]
Thanks for filing! @rosemarie-chiu 

This may not be 100% related, we open it to track the perf issue we talked about on 7/14 from our end and to acknowledge the necessity of improvement. @etr2460 @graceguo-supercat @john-bodley

**Comment by graceguo-supercat** [comment]
this is **not the dashboard load issue** we talked on 7/14. But in airbnb I had similar experience, for some big multiple tabs dashboards, it took very long time to switch tab. this only happens when enable dashboard native filters. 
Please help!

**Comment by PowerPlop** [comment]
@junlincc I experience the same issue when using tabs + native filters in v1.3
The initial load is fast, but when you switch to other tabs the dashboard becomes slower and slower. If you navigate back to the first tab, all viz are slow (noticable when scrolling table viz).
I suspect the browser caches all viz results which is not ideal?

EDIT: Firefox seems to work fine.

**Comment by srinify** [comment]
Converting to Discussion since it's a feature request / area of improvement!
